commands/init.js

```diff
@@ -61,18 +63,22 @@ exports.handler = async (argv) => {
     let assetFileExists = false;
 
     try {
-        log.debug(`Checking for existing "eik.json" file in directory (${cwd})`);
+        log.debug(
+            `Checking for existing "eik.json" file in directory (${cwd})`,
+        );
         try {
             const st = fs.statSync(pathname);
             if (st.isFile()) {
                 assetFileExists = true;
             }
-        } catch(err) {
+        } catch (err) {
             // noop
         }
-        
+
         if (assetFileExists) {
-            throw new Error(`An "eik.json" file already exists in directory. File will not be written`);
+            throw new Error(
+                `An "eik.json" file already exists in directory. File will not be written`,
+            );
         }
```

Comment:
Unrelated to this PR but perhaps this command should make use of the config store util?

Reply:
yea could be